R/shiny.R

```diff
@@ -0,0 +1,35 @@
+#' Open an interactive chat application
+#'
+#' Use `chatApp()` to launch an interactive shiny app for any chat instance.
+#'
+#' @param chat A chat object created by [new_chat_openai()] or friends.
+#' @export
+#' @returns A [shiny::shinyApp].
+chatApp <- function(chat) {
+  check_installed(c("bslib", "shiny", "shinychat"))
+
+  ui <- bslib::page_fluid(
+    shinychat::chat_ui("chat"),
+    shiny::actionButton(
+      "close_btn", "",
+      class = "btn-close",
+      style = "position: absolute; top: 10px; left: 10px;"
```

Comment:
```suggestion
      style = "position: absolute; top: 10px; right: 10px;"
```

Reply:
That makes it overlap with the first chat bubble? (That is also what claude suggested 😉)